tajo-common/src/main/java/org/apache/tajo/conf/TajoConf.java

```diff
@@ -203,6 +203,7 @@ public static int setDateOrder(int dateOrder) {
 
     // Query Configuration
     QUERY_SESSION_TIMEOUT("tajo.query.session.timeout-sec", 60, Validators.min("0")),
+    QUERY_SESSION_CACHE_SIZE("tajo.query.session.cache-size", 1000000, Validators.min("1000000")),
```

Comment:
Thanks for your rapid reply. Here are my answers.
1. Thanks for understanding.
2. I mean, with ```Validators.min("1000000")```, Tajo won't accept _tajo.query.session.query-cache-size-mb=0_. Validators should be used to validate the configuration. So, IMO, ```Validators.min(0)``` is better to prohibit negative values.
3. Thanks for understanding.
4. I also think that query caching is important. I mean, the current implementation will not be popularly used because the cached data can be used only when the exactly same queries are submitted repeatedly.